applications/multiai_ultrasound/operators/visualizer_icardio/CMakeLists.txt

```diff
@@ -23,13 +23,21 @@ add_library(visualizer_icardio SHARED
   visualizer_icardio.cu
   visualizer_icardio.cuh
   visualizer_icardio.hpp
-  )
+)
 
 target_link_libraries(visualizer_icardio
+  PUBLIC
     holoscan::core
-    holoscan::infer
+  PRIVATE
     holoscan::infer_utils
```

Comment:
Just noting that `infer_utils` would be enough if we added a shim for [`raise_error`](https://github.com/nvidia-holoscan/holoscan-sdk/blob/main/modules/holoinfer/src/include/holoinfer_utils.hpp#L59), similar to what is done for [`report_error`](https://github.com/nvidia-holoscan/holoscan-sdk/blob/main/modules/holoinfer/src/include/holoinfer_utils.hpp#L50) with the shim [here](https://github.com/nvidia-holoscan/holoscan-sdk/blob/main/include/holoscan/utils/holoinfer_utils.hpp#L63-L64).